web-wallet/src/routes/(app)/settings/+page.svelte

```diff
@@ -65,6 +65,10 @@
 	let isValidGas = false;
 
 	$: ({ isSyncing } = $walletStore);
+
+	const betaNotice = `These functionalities are currently disabled in
+    the beta version of the web wallet and will be enabled in a future update.`;
```

Comment:
Are we happy with this or should we disable the eslint for this line instead? I have verified it renders as expected.

<img width="815" alt="Screenshot 2024-02-28 at 14 04 32" src="https://github.com/dusk-network/rusk/assets/34094428/f2f9d822-8533-4fbe-aee6-92a81658c1d6">

Reply:
> Are we happy with this or should we disable the eslint for this line instead? I have verified it renders as expected.

Your choice.
If that code is executed once, I usually break the long string in an array for readability, e.g.:

```js
const myLongString = [
    "Lorem ipsum dolor sit amet, consectetur ",
    "adipisicing elit, sed do eiusmod tempor ",
    "incididunt ut labore et dolore magna aliqua."
].join("");
```

If you want to use a "disable" just `// eslint-disable-next-line max-len` instead of switching to backticks and adding an exception about using string templates without variables.

It won't matter in the future as apparently prettier doesn't break strings at all and has the `max-len` setting disabled.

Reply:
It doesn't matter to me either. Also, we will remove this once the features become available so it's a short-term anyways. Maybe we can leave it as I have it?

Reply:
I would expect eslint giving you an error as you have it, without adding an exception.

That's why I said: just disable `max-len` for that line.

Reply:
Done.

Has Kieran managed to find a way to avoid the eslint exception being separated from the code with an empty line?

Reply:
> Done.
> 
> Has Kieran managed to find a way to avoid the eslint exception being separated from the code with an empty line?

What are you referring to?

Reply:
I think he had mentioned that the new lint setup was threatening the `//eslint` rules as comments and was adding an empty line between them and the next line – causing the exception to not be properly applied.

Reply:
Didn't know that (and usually I want a blank line **before** the comment, not after)